lib/libshout-idjc/CMakeLists.txt

```diff
@@ -33,8 +36,39 @@ target_compile_definitions(${PROJECT_NAME}
         LIBSHOUT_MICRO=${PROJECT_VERSION_PATCH}
 )
 
-target_compile_options(${PROJECT_NAME} PRIVATE -pthread -Wall)
-target_link_libraries(${PROJECT_NAME} PRIVATE pthread)
+target_compile_options(${PROJECT_NAME} PRIVATE -Wall)
+
+message("CMAKE_PREFIX_PATH: ${CMAKE_PREFIX_PATH}")
+message("CMAKE_REQUIRED_DEFINITIONS: ${CMAKE_REQUIRED_DEFINITIONS}")
+message("CMAKE_REQUIRED_INCLUDES: ${CMAKE_REQUIRED_INCLUDES}")
+
+cmake_push_check_state()
+
+if (CMAKE_PREFIX_PATH AND NOT CMAKE_PREFIX_PATH STREQUAL "")
+    list(GET CMAKE_PREFIX_PATH 0 PREFIX_PATH)
+    list(APPEND CMAKE_REQUIRED_INCLUDES "${PREFIX_PATH}/include")
+endif()
+
+message("CMAKE_REQUIRED_INCLUDES: ${CMAKE_REQUIRED_INCLUDES}")
+
+# Ensure we have pthreads
```

Comment:
Why isn't this using CMake's built in Threads module?

```
find_package(Threads REQUIRED)
target_link_libraries(${PROJECT_NAME} Threads::Threads)
```

Reply:
Because that defaults to windows threads.